cmd/keycloakb/keycloak_bridge.go

```diff
@@ -236,6 +236,17 @@ func main() {
 	// Keycloak adaptor for common-service library
 	commonKcAdaptor := keycloakb.NewKeycloakAuthClient(keycloakClient, logger)
 
+	// Public Keycloak URL
+	var keycloakPublicURL string
+	{
+		urls := strings.Split(keycloakConfig.AddrTokenProvider, " ")
+		if len(urls) < 1 {
```

Comment:
Is it possible to have 0 items? I think that if keycloakConfig.AddrTokenProvider is empty, Split will return an array with one empty string

Reply:
Good point, I remove it